### DISCUSSION TITLE
如何解决百度地图跨域问题？

### DISCUSSION THREAD

**Discussion by evanzeng** [zouyaoji/vue-cesium#572] [Q&A]
VcImageryProviderBaidu 组件示例显示有跨域问题：
![image](https://github.com/zouyaoji/vue-cesium/assets/872641/ae473436-e622-44b6-bcff-d24e7bf58af6)
请问怎么避免此问题？

**Comment by zouyaoji** [comment]
并不一定是跨域哈，有些瓦片就是没有。 看你截图是完全没显示出来？ 但我这看是有的
